bin/ci/semver.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+function grep_word_from_git_commits_since_last_tag() {
+    local commits=$(git log --oneline $(git describe --tags --abbrev=0 @^)..@ --grep="$1" 2> /dev/null)
+
+    if [ $? -eq 128 ]; then
+        echo 2
+    elif [ -z "$commits" ];
+    then
+        echo 0
+    else
+        echo 1
+    fi
+}
+
+function get_version() {
+    ##
+    # TODO: Comment out this code when packages are stable
```

Comment:
Do we have an issue for this? My experience is that TODOs without issues get forgotten and remain unchanged 🙂.

Reply:
https://github.com/lmc-eu/spirit-design-system/issues/37